src/hotspot/cpu/riscv/templateTable_riscv_64.cpp

```diff
@@ -229,12 +229,24 @@ void TemplateTable::dconst(int value) {
 
 void TemplateTable::bipush() {
   transition(vtos, itos);
-  __ lb(R25_tos, R22_bcp, 1);
+  __ lbu(R25_tos, R22_bcp, 1);
 }
 
 void TemplateTable::sipush() {
   transition(vtos, itos);
-  __ get_2_byte_integer_at_bcp(1, R25_tos, InterpreterMacroAssembler::Signed);
```

Comment:
Can we load a signed byte, shift, load an unsigned byte (to another reg) and then `or` them?
This will require temp register in `get_2_byte_integer`. It would be nice to pass it as parameter instead of using some fixed `rscratchN`